packages/vaex-core/setup.py

```diff
@@ -183,12 +183,14 @@ def __str__(self):
       include_package_data=True,
       ext_modules=([extension_vaexfast] if on_rtd else [extension_vaexfast, extension_strings, extension_superutils, extension_superagg]) if not use_skbuild else [],
       zip_safe=False,
+      python_requires=">=3.6",
```

Comment:
cibuildwheel parses this to determine which wheels to build

Reply:
cc @franz101

see also the [diff above](https://github.com/vaexio/vaex/pull/2331#issuecomment-1399453299)